src/es/mangacrab/src/eu/kanade/tachiyomi/extension/es/mangacrab/MangaCrab.kt

```diff
@@ -47,10 +48,10 @@ class MangaCrab :
         addRandomUAPreferenceToScreen(screen)
     }
 
-    override val pageListParseSelector = "div.page-break:not([style*='display:none'])"
+    override val pageListParseSelector = "div.page-break:not([style*='display:none']) img"
```

Comment:
This will include the 'lee antes en mangacrab (buscanos en google)' image on the first and last image. 

This mirrors what the site does, but if desired you can instead strip these out with:
```suggestion
    override val pageListParseSelector = "div.page-break:not([style*='display:none']) img:not([src])"
```